js/main.js

```diff
@@ -0,0 +1,18 @@
+/* Steps:
+1. Get first number from doc
+2. Get second number from dog
+3. Add the numbers together
+4. Display 
+*/
+
+function addNums(){
+    var numOne = parseFloat(document.getElementById("numberOne").value);
```

Comment:
Use js specific classes when you need to access elements in the DOM, ie `.js-first-number`